test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -831,7 +831,7 @@ def check_hvac_type(prototypes_base)
       run_id = "#{building_type}_#{template}_#{climate_zone}_#{mod_str}"
       @bldg_type_alt_now = @bldg_type_alt[prototype]
 
-      if building_type == 'MidriseApartment' && mod_str == ''
+      if building_type == 'MidriseApartment' && mod_str.nil?
```

Comment:
While testing I noticed that `mod_str == ''` never seem to occur and that `mod_str` was actually `nil` so I made the following changes. Were these cases actually never being run then?

Reply:
I believe they were working when I set them up. Could it be that '' is equivalent to .nil? ?

Reply:
Possibly. Could it also be a change in Ruby? You probably set that up in 2.5, I think that I was using 2.7.2 to run the tests.